include/ts/ts.h

```diff
@@ -2165,11 +2165,40 @@ tsapi void TSDebug(const char *tag, const char *format_str, ...) TS_PRINTFLIKE(2
     @param ... Format arguments.
  */
 tsapi void TSDebugSpecific(int debug_flag, const char *tag, const char *format_str, ...) TS_PRINTFLIKE(3, 4);
-extern int diags_on_for_plugins;
+extern int diags_on_for_plugins; /* Do not use directly. */
```

Comment:
I feel reasonably strongly that for 10.0.0, we should:

1. Mark TSDebug() deprecated here (such that it gives compiler warnings)
2. Change all plugins in the core to use TSDbg() (to avoid warnings from # 1.